### PR TITLE
fixed very low predictions in plots upon discussion with Peter and Brian

### PR DESCRIPTION
**Pull request by brborate**

### CODE REVIEW

File: cor_surrogates/code/.nfs0000000006dc75ee000066e8 [hunk 1]
```diff
@@ -0,0 +1,291 @@
+#-----------------------------------------------
```
Comment (benkeser) [inline cor_surrogates/code/.nfs0000000006dc75ee000066e8:1]: What’s up with this file name?

Comment (brborate) [reply cor_surrogates/code/.nfs0000000006dc75ee000066e8]: 👍 ya, for some reason this didn't get deleted right away. Had to log off rhino and log back in and it was magically gone.

**Comment by benkeser** [comment]
Could you please check the diff and make sure this PR is accomplishing what you hope? I don’t see the code changes that would align with the title of the PR. Am I missing something?

**Comment by brborate** [comment]
> Could you please check the diff and make sure this PR is accomplishing what you hope? I don’t see the code changes that would align with the title of the PR. Am I missing something?

The change primarily was in the plot_predicted_probabilities function wherein we were taking weighted predictions, which lowered the prediction probabilities. Now we are taking the predictions as they are (not weighting them) which fixes the plots. 👍

**Pull request merged**